sqlx-core/src/sqlite/statement/handle.rs

```diff
@@ -279,3 +281,55 @@ impl StatementHandle {
         Ok(from_utf8(self.column_blob(index))?)
     }
 }
+
+unsafe fn wait_for_unlock_notify(conn: *mut sqlite3) -> Result<(), SqliteError> {
```

Comment:
I think there must have been a rebase error somewhere? This commit only adds function declarations but doesn't call them anywhere ...

Reply:
hmm I dont think we can cherry pick these onto 0.5.0. They have changed how the statements are executed and these commits depend on that.